src/js/contentUtils.ts

```diff
@@ -130,79 +141,114 @@ export function storeFoundJS(scriptNodeMaybe: HTMLScriptElement): void {
       return;
     }
 
+    if (rawManifest === null) {
+      return;
```

Comment:
Should probably update the state to invalid here.

Reply:
If I understand correctly, once we go to INVALID we can never go back to any other state... I'll move this null check inside the `try` so that it gets handled by the retry logic.